lib/config.ts

```diff
@@ -20,6 +20,12 @@ export function isAppEnabled(app: 'news' | 'notes' | 'photos') {
   return enabledApps.includes(app);
 }
 
+export function isCookieDomainAllowed(domain: string) {
+  const allowedDomains = (Deno.env.get('CONFIG_ALLOWED_COOKIE_DOMAINS') || '').split(',') as typeof domain[];
+
+  return allowedDomains.includes(domain);
```

Comment:
If I'm reading this correctly, it'll reject any domain if `CONFIG_ALLOWED_COOKIE_DOMAINS` isn't set. Since we want to make this optional, we should allow this to pass if there are no allowed cookie domains.

```suggestion
  const allowedDomains = (Deno.env.get('CONFIG_ALLOWED_COOKIE_DOMAINS') || '').split(',') as typeof domain[];
  
  if (allowedDomains.length === 0) {
    return true;
  }

  return allowedDomains.includes(domain);
```

Reply:
Wow, nice catch!